docs/en/brand-store.md

```diff
@@ -0,0 +1,198 @@
+          administer your Brand Store and manage model specific activities such
+          as signing the model and uploading the model’s custom gadget snap.
+          Using a single Brand SSO account simplifies things and is therefore
+          recommended in many situations.
+
+You use a Brand SSO account to create keys used for brand activities:
+
+![image][brand-store-2]
+
+## Assertions
+
+[Assertions][assertions] are signed pieces of information. That is, an
+assertion provides data, and the entire statement is signed by a key bound to
+an Ubuntu SSO account. This allows the source of the information to be
+authenticated and the information it contains to be validated.
+
+The snap system uses assertions in many places. For example, the model 
+assertion (discussed next) provides key information defining the device **model**. 
+It is signed by a key under a Brand SSO account. This means it can be
+ authenticated and the data it contains can be relied on as correct.
+
+## Models
+
+Every system that is connected to a Brand Store contains a signed **model 
+assertion**.
+
+!!! Note:
+          Any system that does not have a customized model assertion cannot use
+          a Brand Store. (Classic systems with no custom model assertion now
+          have a generic one that points at the Ubuntu Store.)
+
+The model assertion contains data such as:
+
+- The Brand Store it points to
+- The model name
+- The Brand Store SSO account-id
+- The gadget snap name (and on Core systems, the kernel snap)
+- And more
+
+As noted, the model assertion is **signed** by a key generated under the Brand
+ Store SSO account and is placed in an image. The image is then ready to go to
+production and distribution. (When you create an image using the `ubuntu-image` 
+tool, the model assertion and various other required assertions are inserted
+ into the image automatically. If you manually create an image, you need to 
+take these steps yourself. See [Seeding a Classic
+ Image][seeding-a-classic-image].)
+
+!!! Note:
+          As mentioned above, it is strongly recommended to use the Brand Store
+          SSO account to generate the key that signs the model assertion. This
+          document assumes you use the Brand Store SSO account for all such
+          purposes.
+
+Using a Brand SSO account key to sign a model used in an image:
+
+![image][brand-store-3]
+
+## Systems pointing at the Brand Store
+
+Here we overview what has been discussed so far:
+
+- Systems contain a model assertion.
+- The model assertion contains the account-id of the Brand SSO account and
+was signed by a key made under that account.
+- The model assertion points at the Brand Store.
+
+!!! Note:
+          This picture still does not show the required system authentication
+          through which a system may gain access to their Brand Store.
+
+Systems whose model points at a Brand Store are rejected unless authentication
```

Comment:
Are systems people or things? Feels awkward.

Reply:
should be "its". I don't have a problem with the anthropomorphism here. One can say: here is my car. its tire is flat.